docs/faq.md

```diff
@@ -149,3 +149,11 @@ server into multiple *nodes*, each with one GPU. In this case the
 orchestration environment will run a different Triton for each GPU and
 an load balancer will be used to divide inference requests across the
 available Triton instances.
+
+## If the server segfaults, how can I debug it?
+
+The NGC build is a Release build and does not contain Debug symbols. 
+The build.py as well defaults to a Release build. Refer to the instructions
+in [build.md](build.md#building-with-debug-symbols) to create a Debug build
+of Triton. This will help find the cause of the segmentation fault when
+looking at the gdb trace for the segfault.
```

Comment:
In most of the cases the users won't be able to resolve the issue themselves.. debug builds only help them to locate the point of failure. We can add another line for asking them to include the backtrace when opening a seqfault issue on Triton.

Reply:
Good idea. Added a line for the same.